docs/userguide/roleseparation/federated-resource-quota.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Federated ResourceQuota
+---
+
+## Background
+
+With the widespread used of multi-clusters, the single-cluster quota management `ResourceQuota` in Kubernetes can no longer meet the administrator's resource management and restriction requirements for federated clusters. 
+Resource administrators often need to stand on the global dimension to manage and control the total consumption of resources by each business.
+
+Creating a corresponding `namespace` and `ResourceQuotas` under each Kubernetes cluster is the usual practice, and then Kubernetes will limit the resources by `ResourceQuotas`. 
+However, with the growth of the number of businesses, the expansion and contraction of member clusters, the number of available resources and resource types of each cluster are different, which brings many problems to the administrator.
+
+In addition, Karmada supports the propagation of ResourceQuota objects to Kubernetes clusters through PropagationPolicy, so as to achieve the purpose of creating quotas on multi-clusters by kubernetes native APIs. 
+However, it is impossible to freely adjust and limit the global resource usage of a business by PropagationPolicy, and it is quite troublesome to expand such a requirement on it. 
+We need a global quota for Karmada, not just ResourceQuota on member clusters.
+
+The `FederatedResourceQuota` API is provided to solve the above problems. Here we give several typical usage scenarios of `FederatedResourceQuota`.
+
+## What can FederatedResourceQuota do
+
+FederatedResourceQuota supports:
+
+* The global quota management for multiple clusters managed by Karmada.
+* Fine-grained management of quotas under the same namespace of different clusters.
+    * Ability to enumerate resource usage limits per namespace.
+    * Ability to monitor resource usage for tracked resources.
+    * Ability to reject resource usage exceeding hard quotas.
+
+![unified resourcequota](../../resources/key-features/unified-resourcequota.png)
+
+FederatedResourceQuota supports the management of the following resources: cpu, memory, storage and ephemeral-storage.
+
+## Deploy a simplest FederatedResourceQuota
+
+For example, as an administrator, you want to deploy business A across multiple clusters under namespace `test`.
+
+You can create a namespace called test on the Karmada control plane. Karmada will automatically create the corresponding namespace in the member clusters.
+
+```shell
+kubectl --kubeconfig ~/.kube/karmada.config --context karmada-apiserver create ns test
+```
+
+You want to create a resource limit of 100 core CPUs for business A.
+The available resources on clusters:
+
+* member1: 20C
+* member2: 50C
+* member3: 100C
```

Comment:
This number seems incorrect.

Reply:
The numbers here are allocatable resources.